contributors/devel/conformance-guidelines.md

```diff
@@ -0,0 +1,83 @@
+# Conformance Test Guidelines
+
+The Kubernetes conformance test suite is a set of testcases, currently a
+subset of the integration/e2e tests, that the Architecture SIG has approved
+to define the core set of interoperable features that all Kubernetes
+deployments must support.
+
+Conformance tests in the default profile should cover features that are:
+
+- GA (not alpha or beta APIs, nor deprecated features)
+- portable (not dependent on provider-specific capabilities or on the public internet)
+- not an optional feature (e.g. not policy enforcement)
+- non-privileged (neither root on nodes, network, nor cluster)
+
+For each Kubernetes release, a Conformance Document will be generated
+that lists all of the tests that comprise the conformance test suite, along
+with the formal specification of each test. This document will help people
+understand what features are being tested without having to look through
+the testcase's code directly.
+
+## Adding New Tests
+
+To include a testcase in the conformance test suite, the following
+steps must be taken:
+- the testcase must use the `framework.ConformanceIt()` function rather
+  than the `framework.It()` function
+- the testcase must include a comment immediately before the
+  `framework.ConformanceIt()` call that includes all of the required
+  metadata about the test (see the [Test Metadata](#test-metadata) section)
+
```

Comment:
Can you also include these guidelines for the PR to promote a test to Conformance

* Please use "Promote xxx e2e test to Conformance" as template of your PR title
* Tag your PR with "/area conformance" label
* Send your PR to Sig-Architecture for review by adding "@kubernetes/sig-architecture-pr-reviews"
* CC your Sig, Sig-Architecture and me in your PR